dashboard/app/models/lesson.rb

```diff
@@ -229,6 +229,11 @@ def summarize(include_bonus_levels = false, for_edit: false)
     lesson_summary = Rails.cache.fetch("#{cache_key}/lesson_summary/#{I18n.locale}/#{include_bonus_levels}") do
       cached_levels = include_bonus_levels ? cached_script_levels : cached_script_levels.reject(&:bonus)
 
+      description_student = I18n.t("data.script.name.#{script.name}.lessons.#{key}.description_student", default: '')
```

Comment:
We prefer

```suggestion
      description_student = I18n.t(
        'description_student',
        scope: [:data, :script, :name, script.name, :lessons, key],
        smart: true,
        default: ''
      )
```

for strings whose scopes include dynamic content (script name, lesson key) for which it's possible to include the `.` character

Reply:
I just copied over the old access. I'll fix it while I'm here though.